src/layouts/BaseLayout/BaseLayout.tsx

```diff
@@ -70,7 +82,7 @@ function BaseLayout({ children }) {
     <>
       <Suspense fallback={<FullPageLoader />}>
         <RepoBreadcrumbProvider>
-          {isFullExperience ? (
+          {isFullExperience || isImpersonating ? (
```

Comment:
this is to prevent the weird case when impersonating someone who has not completed onboarding